src/components/Cards.js

```diff
@@ -102,7 +116,7 @@ const Title = styled.h2`
   margin: 0;
 `;
 
-const Header = styled.div`
+const Row = styled.div`
```

Comment:
I would say the name `Header` would be better to understand then `Row`. You can also work with `header` instead of `div`